integration/test_auth.py

```diff
@@ -32,11 +33,18 @@
 from firebase_admin import credentials
 
 
-_verify_token_url = 'https://www.googleapis.com/identitytoolkit/v3/relyingparty/verifyCustomToken'
-_verify_password_url = 'https://www.googleapis.com/identitytoolkit/v3/relyingparty/verifyPassword'
-_password_reset_url = 'https://www.googleapis.com/identitytoolkit/v3/relyingparty/resetPassword'
-_verify_email_url = 'https://www.googleapis.com/identitytoolkit/v3/relyingparty/setAccountInfo'
-_email_sign_in_url = 'https://www.googleapis.com/identitytoolkit/v3/relyingparty/emailLinkSignin'
+_EMULATOR_HOST_ENV_VAR = 'FIREBASE_AUTH_EMULATOR_HOST'
```

Comment:
I suppose the question here is whether the knowledge that some tests do pass worth marking the failing tests as being skipped for emulator (they should still be run for the "real" integration tests). At the least, this does reduce future work a bit since we'll already know which tests to go about fixing, and *might* catch a breakage in the meantime.

Reply:
It's a lot of knowledge to capture in one PR. So let's move that into a separate PR, and keep this one focused on the implementation.

We also encountered some flaky tests when we tried this in Node.js, which is why that PR is still on hold. So a passing integration test doesn't necessarily mean everything is ok either.